src/file.ts

```diff
@@ -530,7 +530,7 @@ class File extends ServiceObject {
         {
           method: 'POST',
           uri: `/rewriteTo/b/${destBucket.name}/o/${
-              encodeURIComponent(destName)}`,
+              encodeURIComponent(newFile.name)}`,
```

Comment:
The `File` constructor already parses out extra slashes from a given object name, so we can just re-use that here.